packages/@divvi/mobile/src/earn/EarnDepositConfirmationScreen.tsx

```diff
@@ -112,8 +128,144 @@ export default function EarnDepositConfirmationScreen({ route: { params } }: Pro
               apy: getTotalYieldRate(pool).toFixed(2),
             })}
           />
+
+          <SwapAndDepositSummaryItem
```

Comment:
nit: instead of passing a mode and returning `null`, could  just do `mode === 'swap-deposit' && <SwapAndDepositSummaryItem ... />`